models/UserModel.ts

```diff
@@ -134,6 +134,7 @@ export class UserModel extends BaseEntity {
       bio: this.bio,
       points: this.points,
       credits: this.credits,
+      resumes: this.resumes,
```

Comment:
Do we need this? Since all resumes should be handled through the controller, I feel like we don't need a field for this in the UserModel. The "Resumes" table and controller should handle all of the Resume viewing/updating for a single user

Reply:
I think this is fine - getting a user's own profile should return all the info regarding that user, which includes their resume(s). Resumes are tightly related to a user's profile (since that's one of the only places they're displayed), so I don't particularly see an issue with having them returned in the same API call.

Also, @dowhep I don't believe this code works here because we aren't [eager-loading](https://typeorm.io/eager-and-lazy-relations) resumes. This means that this field will be undefined every time we make this function call. Instead, we need to query the Resumes table for all resumes for that user, and attach it to this field. Then, we can return the fields as you specified.

Reply:
Apologies for revisiting such an old comment, but re-visiting this code I don't think what I recommended here is actually good practice. I think having a field here un-explicitly always returning `null` is something very easy to look over, so instead, I'd rather us just outright not specify the field here and instead make `resumes` optional in `PrivateProfile`. That way, we can return this object without the `resumes` field, and we can continue to add it as we do in `UserAccountService.getFullUserProfile`. I think this way gets rid of any ambiguity the code has by always returning `null`.

We can also remove the comment above mentioning that "resumes is null by design".